packages/components/form/src/form-item.vue

```diff
@@ -178,26 +178,26 @@ const fieldValue = computed(() => {
 })
 
 const normalizedRules = computed(() => {
-  const { required } = props
+  const { required, prop } = props
 
   const rules: FormItemRule[] = []
 
-  if (props.rules) {
+  if (props.rules && prop) {
```

Comment:
Maybe there is no need to add extra conditions because the `prop` here must not be `undefined`.

Reply:
Done.